keps/sig-instrumentation/1602-structured-logging/README.md

```diff
@@ -586,15 +586,26 @@ _This section must be completed when targeting alpha to a release._
   - [ ] Feature gate (also fill in values in `kep.yaml`)
     - Feature gate name:
     - Components depending on the feature gate:
-  - [x] Other
-    - Describe the mechanism: New flag `--logging-format` will be used to change logging format of component
-    - Will enabling / disabling the feature require downtime of the control
-      plane? Switching between log formats will require restarting control plane
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled). Switching between log formats will require kubelet restart
+  - [x] **Other**
+    - **Describe the mechanism:**
+
+      New flag `--logging-format` will be used to change logging format of components. This flag will become standard
+      flag for all k8s components and will be added to `k8s.io/component-base`.
+      For alpha we are planning to add it to: `kube-apiserver`, `kube-scheduler`, `kubelet`, `kube-controller-manager`
+
+    - **Will enabling / disabling the feature require downtime of the control
+      plane?**
+
+      Switching between log formats will require restarting control plane
+
+    - **Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).**
+
+      Switching between log formats will require kubelet restart
```

Comment:
I guess the same for things like kube-proxy or any other agents.

So maybe:

... require restarting all node components for which we want to switch the format.